selectrum.el

```diff
@@ -1606,7 +1606,7 @@ Otherwise, just eval BODY."
 (cl-defun selectrum-read
     (prompt candidates &rest args &key
             default-candidate initial-input require-match
-            history no-move-default-candidate
```

Comment:
Why are you modifying the API? Is this not yet part of the released API or are you gradually downstripping the API (I am all for that!)

Reply:
Well you are too fast as always :D

Reply:
Sorry, I should probably not watch this repository ;) Maybe I switch my watching mode to only watch branches/issues I am involved in. But this wouldn't have prevented me from looking here. Actually I just wanted to see if you already merged this, such that I should adapt Consult.

Reply:
No problem, I'm not used to someone watching so sometimes I'm pushing intermediate states but I'm starting to adapt ;)

Reply:
So I will ignore the option for now, so this won't break any code but they would have to update it. We could also wait and then do all breaking changes at once, would you prefer that?

Reply:
I already answered independently, yes I would keep everything as is. And then just deprecate selectrum-read at once if this is what you are up to. I usually try to keep changes somehow separated instead of merging multiple things, but maybe my chaotic development style on the consult repo with many small commits suggests otherwise. Let's say I at least try to do it when implementing major features or large refactorings.

Reply:
I will continue in this PR to make selectrum-read private then. My hope is that not to many use it, the longer the wait the more could be affected anyway.

Reply:
Nah I will make it private in another PR and then continue with this one.

Reply:
@minad I'm also glad to get some feedback, especially when as valuable as yours. I just have to change my habits a bit, I'm very used to push things once in a while, and I'm trying things along the way. When I add a changelog that tells usually that things are in a state how I would like to submit them.